### PR TITLE
Ansible - added users flows

### PR DESCRIPTION
**Pull request by BoicuAlexandra**
added:
- create_user
- delete_user 
- get_user_id
- list_users

### CODE REVIEW

File: content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        default: 'false'
+        required: false
+    - x_509_hostname_verifier:
+        default: strict
+        required: false
+    - worker_group:
+        default: RAS_Operator_Path
+        required: false
+  workflow:
+    - create_new_user:
```
Comment (patmante) [inline content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl:103]: Assign "worker_group" for each step to all flows
image

**Review by patmante — CHANGES_REQUESTED** [review]

File: content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+########################################################################################################################
+#!!
+#! @description: This flow will create a new User object in your Ansible Tower system.
+#!
+#! @input ansible_automation_platform_url: Ansible Tower API URL to connect to (example: https://192.168.10.10/api/v2)
```
Comment (patmante) [inline content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl]: use "the Ansible Automation Platform"  instead of Ansible Tower

**Review by patmante — CHANGES_REQUESTED** [review]

File: content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl (hunk 2 above)
Comment (BoicuAlexandra) [reply content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl]: Done

File: content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl (hunk 1 above)
Comment (BoicuAlexandra) [reply content/io/cloudslang/redhat/ansible/automation_platform/users/create_user.sl:103]: done

**Review by patmante — APPROVED** [review]

**Review by Sailakshith6 — APPROVED** [review]

**Pull request merged**